.github/workflows/test.yml

```diff
@@ -1,8 +1,9 @@
 name: Run tests
 on:
   workflow_dispatch:
+  push:
```

Comment:
This will run CI when semantic-release commit changes related to release (updated version in `package.json` and generated changelog). I would like to avoid that. Since the branch is protected that there is no way to push commit directly, checks on PR is enough to ensure code quality.

Reply:
AFAIK until yesterday, this branch was not protected.

And adding this will actually run the CI AFTER stuff is pushed - so that you can easily see the status @HEAD.

Reply:
Keep it there, then.